Docker/centralserver/Dockerfile

```diff
@@ -91,5 +91,4 @@ COPY files/cs-entrypoint.sh /root/entrypoint.sh
 COPY --chown=root:root files/cs-xroad.conf /etc/supervisor/conf.d/xroad.conf
 CMD ["/root/entrypoint.sh"]
 
-VOLUME ["/etc/xroad", "/var/lib/xroad", "/var/lib/postgresql/16/main/"]
-EXPOSE 4000 2222
```

Comment:
Although, EXPOSE is not explicitly required, it's sometimes good to have it for documentation.